ansible/roles/configure-batyam/README.md

```diff
@@ -0,0 +1,36 @@
+Configure Batyam
```

Comment:
This role is about rolling out batyam. Not configuring Batyam.
name the role 'batyam'

Reply:
Will do